src/cards/productive-time-card.ts

```diff
@@ -39,10 +57,14 @@ const getProductiveTimeData = async function (username: string, utcOffset: numbe
     // process productiveTime
     const chartData = new Array(24);
     chartData.fill(0);
+    // eslint-disable-next-line prefer-const
```

Comment:
If possible, I think we should avoid disabling the `eslint` here. Could we do that? 🤔